### DISCUSSION TITLE
change retroachievements SFX

### DISCUSSION THREAD

**Discussion by quirkunlimited** [forkymcforkface/RGBPi-Extra#17] [Q&A] [answered]
First of all thank you for this! I just started using it.
I executed that script and it made no difference. retroachievements still makes that extremely annoying sound. I'm not sure I'm using it correctly.  
I couldn't change it manually because I can't get root access.
BTW the video on boot is amazing too! thank you!

**Comment by forkymcforkface** [comment, marked as answer]
@quirkunlimited fixed it! if you want to download just the new bash you can put it into the scripts folder. No idea why it had the restore retroarch script code in there. 

https://github.com/forkymcforkface/RGBPi-Extra/blob/main/RGBPi-Extra/application/data/scripts/Retroarch%20-%20Change%20achievement%20sfx.bash

**Reply by quirkunlimited** [reply]
wow! I can't wait to try it out! Thank you again for this tool. I'm having a lot of fun with it.
